### ISSUE TITLE
Dragging button to set position is not working.

### ISSUE THREAD

**Issue by ThiagoCururu** [lemasato/POE-Trades-Companion#156]
As title, dragging buttons to customize its position is not working on Windows 10 pro 1803 build 17134.228. Companion is on 1.13.3 version. Thanks for the amazing tool!

**Comment by lemasato** [comment]
Are you using an OS scaling different than 100%?  
![image](https://user-images.githubusercontent.com/14184969/45332193-798bc300-b56f-11e8-9706-293a1ccdcd0e.png)

**Comment by ThiagoCururu** [comment]
It was at 100%, but I just figured now that it needs an empty space to move it as I imagined it would change places with the other button. Sorry for wasting your time :O

Or while we are at it, I believe there must be other users with the same issue but they won't post here: Is it possible to add a text explaining that an empty slot is needed? Example "Drag to set its position (needs a empty/hidden slot)"

Thanks for your amazing tool and support!
